_layouts/homepage.html

```diff
@@ -21,12 +21,18 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
       <em>vulekamali</em> is a project by the South African National Treasury and Imali Yethu with the goal of making government budget data and processes accessible to all citizens & interested parties. <a href="/about">Read more about vulekamali.</a>
     </div>
 
-    <div class="u-marginBottom25">
-      {%
-        include components/universal/Alert/index.html
-        title="Please note"
-        message="...pending description"
-      %}
+    <div class="Alert">
+      <div class="Alert-content">
+        {%
+          include components/universal/Icon/index.html
+          type="alert"
+          size="large"
+        %}
+        <span class="Alert-heading">
+          Please note
+        </span>
+      </div>
+      <div class="Alert-text">The full Medium Term Budget Policy Statement is <a href="#">available on National Treasury's website</a>. The MTBPS documents and data will be added to vulekamali.gov.za soon.</div>
```

Comment:
Great. This link should be the following, if they stick to their old scheme:

`http://www.treasury.gov.za/documents/mtbps/2018/default.aspx`